templates/zerver/help/communities-directory.md

```diff
@@ -0,0 +1,30 @@
+# Zulip communities directory
+
+{!communities-directory-intro.md!}
+
+## Give permission to be in the Zulip communities directory
+
+{!admin-only.md!}
+
+{start_tabs}
+
+{settings_tab|organization-profile}
+
+1. Toggle **Advertise organization in the Zulip communities
```

Comment:
This article should mention that the server needs to have enabled the public access option, as it is in Zulip Cloud, for this option to be enabled.